app/interfaces/api/entities/search_result.rb

```diff
@@ -39,6 +39,7 @@ class SearchResult < BaseEntity
         expose :supplementary
         expose :agfs_hardship
         expose :lgfs_hardship
+        expose :scheme_12_warning
```

Comment:
[consistent naming](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r476385484)

Reply:
👍 have amended naming to scheme_12_fees_warning